services/bls_aggregation/blsagg_test.go

```diff
@@ -761,6 +773,126 @@ func TestBlsAgg(t *testing.T) {
 	})
 }
 
+func TestIntegrationBlsAgg(t *testing.T) {
```

Comment:
This test is unreadable. unclear what's happening: probably better to divide/refactor
- setup
- what is actually being tested
- add comments maybe?

Reply:
refactored in [ba79798](https://github.com/Layr-Labs/eigensdk-go/pull/277/commits/ba797983f71fd442c7a2c197cfb60f6ba1a03e6c). Check now, hopefully its easier to read :D